README.md

```diff
@@ -33,46 +33,9 @@ with the latest OSM data. For speed and for deterministic tests, there are also
 automatically use the latest OSM data. These are manually managed and hosted by
 Dustin on assets.od2net.org.
 
-If you're developing locally, you can avoid hitting od2net.org by setting up
-three directories in `web/public/`: `osm`, `boundaries`, and `cnt_demand`. If
-`web/public/osm/areas.json` exists, then the Svelte app will load from
-localhost, not od2net.org. You can copy from od2net.org to set this up,
-choosing what study areas to cache:
-
-```
-AREAS="bristol edinburgh strasbourg ut_dallas"
-
-cd web/public
-mkdir boundaries osm cnt_demand
-
-cd osm
-wget https://assets.od2net.org/severance_pbfs/areas.json
-for x in $AREAS; do
-  wget https://assets.od2net.org/severance_pbfs/$x.pbf
-done
-
-cd ../boundaries
-for x in $AREAS; do
-  wget https://assets.od2net.org/boundaries/$x.geojson
-done
-cd ..
-```
-
-There are three more directories particular to Scotland. To cache all of that data:
-
-```
-# Still in web/public
-
-mkdir cnt_osm cnt_boundaries cnt_demand
-jq '.features[] | .properties.kind + "_" + .properties.name' ../../data_prep/scotland/boundaries.geojson | sed 's/"//g' | while read x; do
-  wget https://assets.od2net.org/cnt_boundaries/$x.geojson
-  wget https://assets.od2net.org/cnt_osm/$x.osm.pbf
```

Comment:
Some of the `$x` values had spaces, so this didn't work as copy/paste.

And then the following line 404'd. 

I'm not sure if it's a typo in the path or if the demand models just haven't been updated yet.

These two things are what inspired me to formalize this into an actually executable script.